exercises/go-counting/go_counting_test.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import gocounting
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
```

Comment:
Minor change: please add 1 additional blank line above this (2 before, 1 after) for consistency across exercises.